ironfish-cli/src/commands/reset.ts

```diff
@@ -80,6 +80,32 @@ export default class Reset extends IronfishCommand {
     this.sdk.internal.set('isFirstRun', true)
     await this.sdk.internal.save()
 
+    const node = await this.sdk.node()
+    const walletDb = node.wallet.walletDb
+
+    const walletDbStores: IDatabaseStore<{
+      key: Readonly<unknown>
+      value: unknown
+    }>[] = [
+      walletDb.decryptedNotes,
+      walletDb.nullifierToNoteHash,
+      walletDb.sequenceToNoteHash,
+      walletDb.nonChainNoteHashes,
+      walletDb.transactions,
+      walletDb.sequenceToTransactionHash,
+      walletDb.pendingTransactionHashes,
+      walletDb.timestampToTransactionHash,
+      walletDb.assets,
+      walletDb.nullifierToTransactionHash,
+      walletDb.unspentNoteHashes,
+    ]
+
```

Comment:
this list isn't reused anywhere right? was wondering if it's worth setting it up as a constant (or something like that) somewhere, but if this is the only place it's used, no point